### ISSUE TITLE
Issue about using the inpaint network to train a model for pictures with smaller size

### ISSUE THREAD

**Issue by Zhong-gy** [JiahuiYu/generative_inpainting#351]
Hi @JiahuiYu, thanks for your work.
I am a freshman in neural network. 
At present, I am tring to using your model directly to train a model for the picture inpainting( The size of pictures is 32x32(or 16x16), and the inpaint block is 16x16(or 8x8 if the picture size is16x16) located in the bottom-right of the picture ), using DeepFill v1. The generative loss converges very quickly at first but it seems almost unchanged after several hours with gtx2080 (Even days of training, still almost unchanged) and the inpaint result is kind of bad... My training data is DTD dataset, and I crop it to smaller size(32x32) before training. The whole dataset contains about 4000000 pictures at last.
Do you have any Suggestions about this? Sorry to bother you, I was so puzzled what to do now. If I need to modify the model for this problem, how should I get started?

**Comment by JiahuiYu** [comment]
@Zhong-gy Hi thanks for your interest in our work.

If you image is only 32x32, I'd suggest to modify the network architecture (e.g., reduce number of layers and the downsampling blocks).
